### PR TITLE
Rename props `isDisabled` to `disbled` in `<Label />`

### PR DESCRIPTION
**Pull request by Andresbl123**
The `isDisabled` property is renamed to `disbled` in the `<Label />`component. This change makes the code easier to read and conforms to the design system standards.

**BREAKING CHANGE**
This name change may cause the component to not behave as expected in the places where it is used. It is **important to validate and adjust the code so that the component** works correctly with the new property name.

### CODE REVIEW

**Review by cmarin001 — COMMENTED** [review]
revisar todos los lugares donde se esta usando el componente 
<Label/>
image

File: src/components/inputs/Label/stories/Label.Size.stories.tsx [hunk 1]
```diff
@@ -35,7 +36,7 @@ const Size = ({
     >
       {typos.map((typo) => (
         <Label
-          isDisabled={isDisabled}
+          disabled={isDisabled}
```
Comment (cmarin001) [inline src/components/inputs/Label/stories/Label.Size.stories.tsx:39]: sigue siendo isDisabled

Comment (Andresbl123) [reply src/components/inputs/Label/stories/Label.Size.stories.tsx]: Resuelto

**Review by cmarin001 — APPROVED** [review]
cumple con los requerimientos

**Pull request merged**